*** ./smali/android/preference/PreferenceFrameLayout$LayoutParams.smali	2013-10-12 21:22:31.956391529 +0800
--- /home/santajin/MIUI_42/i9100/temp/new_smali/framework.jar.out/./smali/android/preference/PreferenceFrameLayout$LayoutParams.smali	2013-10-12 21:25:56.252401416 +0800
***************
*** 59,64 ****
--- 59,72 ----
  
      iput-boolean v1, p0, Landroid/preference/PreferenceFrameLayout$LayoutParams;->removeBorders:Z
  
+     iget-boolean v1, p0, Landroid/preference/PreferenceFrameLayout$LayoutParams;->removeBorders:Z
+ 
+     invoke-static {p1, v1}, Landroid/preference/PreferenceFrameLayout$Injector;->isRemoveBorders(Landroid/content/Context;Z)Z
+ 
+     move-result v1
+ 
+     iput-boolean v1, p0, Landroid/preference/PreferenceFrameLayout$LayoutParams;->removeBorders:Z
+ 
      invoke-virtual {v0}, Landroid/content/res/TypedArray;->recycle()V
  
      return-void
