*** ./smali/android/preference/RingtonePreference.smali	2013-10-12 21:22:31.940391527 +0800
--- /home/santajin/MIUI_42/i9100/temp/new_smali/framework.jar.out/./smali/android/preference/RingtonePreference.smali	2013-10-12 21:25:56.212401414 +0800
***************
*** 311,316 ****
--- 311,322 ----
  
      invoke-virtual {p1, v0, v1}, Landroid/content/Intent;->putExtra(Ljava/lang/String;Ljava/lang/CharSequence;)Landroid/content/Intent;
  
+     const-string v0, "com.android.thememanager"
+ 
+     const-string v1, "com.android.thememanager.activity.ThemeTabActivity"
+ 
+     invoke-virtual {p1, v0, v1}, Landroid/content/Intent;->setClassName(Ljava/lang/String;Ljava/lang/String;)Landroid/content/Intent;
+ 
      return-void
  .end method
  
