*** ./smali/android/view/WindowManagerPolicy.smali	2013-10-12 21:22:51.120392456 +0800
--- /home/santajin/MIUI_42/i9100/temp/new_smali/framework.jar.out/./smali/android/view/WindowManagerPolicy.smali	2013-10-12 21:26:17.616402449 +0800
***************
*** 22,27 ****
--- 22,29 ----
  
  .field public static final ACTION_PASS_TO_USER:I = 0x1
  
+ .field public static final ACTION_POKE_USER_ACTIVITY:I = 0x2
+ 
  .field public static final ACTION_WAKE_UP:I = 0x2
  
  .field public static final EXTRA_HDMI_PLUGGED_STATE:Ljava/lang/String; = "state"
