*** ./smali/com/android/internal/telephony/IccPhoneBookInterfaceManagerProxy.smali	2013-10-12 21:22:05.192390234 +0800
--- /home/santajin/MIUI_42/i9100/temp/new_smali/telephony-common.jar.out/./smali/com/android/internal/telephony/IccPhoneBookInterfaceManagerProxy.smali	2013-10-12 21:25:16.692399502 +0800
***************
*** 83,88 ****
--- 83,97 ----
      return-object v0
  .end method
  
+ .method getIccPhoneBookInterfaceManager()Lcom/android/internal/telephony/IccPhoneBookInterfaceManager;
+     .locals 1
+ 
+     .prologue
+     iget-object v0, p0, Lcom/android/internal/telephony/IccPhoneBookInterfaceManagerProxy;->mIccPhoneBookInterfaceManager:Lcom/android/internal/telephony/IccPhoneBookInterfaceManager;
+ 
+     return-object v0
+ .end method
+ 
  .method public setmIccPhoneBookInterfaceManager(Lcom/android/internal/telephony/IccPhoneBookInterfaceManager;)V
      .locals 0
      .parameter "iccPhoneBookInterfaceManager"
