*** ./smali/com/android/internal/telephony/ServiceStateTracker.smali	2013-10-12 21:22:04.568390202 +0800
--- /home/santajin/MIUI_42/i9100/temp/new_smali/telephony-common.jar.out/./smali/com/android/internal/telephony/ServiceStateTracker.smali	2013-10-12 21:25:15.720399455 +0800
***************
*** 883,888 ****
--- 883,897 ----
      return v0
  .end method
  
+ .method public getIccRecords()Lcom/android/internal/telephony/IccRecords;
+     .locals 1
+ 
+     .prologue
+     iget-object v0, p0, Lcom/android/internal/telephony/ServiceStateTracker;->mIccRecords:Lcom/android/internal/telephony/IccRecords;
+ 
+     return-object v0
+ .end method
+ 
  .method protected abstract getPhone()Lcom/android/internal/telephony/Phone;
  .end method
  
