*** ./smali/android/net/CaptivePortalTracker.smali	2013-10-12 21:22:24.952391189 +0800
--- /home/santajin/MIUI_42/i9100/temp/new_smali/framework.jar.out/./smali/android/net/CaptivePortalTracker.smali	2013-10-12 21:25:45.460400894 +0800
***************
*** 9,15 ****
          Landroid/net/CaptivePortalTracker$DelayedCaptiveCheckState;,
          Landroid/net/CaptivePortalTracker$ActiveNetworkState;,
          Landroid/net/CaptivePortalTracker$NoActiveNetworkState;,
!         Landroid/net/CaptivePortalTracker$DefaultState;
      }
  .end annotation
  
--- 9,16 ----
          Landroid/net/CaptivePortalTracker$DelayedCaptiveCheckState;,
          Landroid/net/CaptivePortalTracker$ActiveNetworkState;,
          Landroid/net/CaptivePortalTracker$NoActiveNetworkState;,
!         Landroid/net/CaptivePortalTracker$DefaultState;,
!         Landroid/net/CaptivePortalTracker$Injector;
      }
  .end annotation
  
***************
*** 885,893 ****
  
      invoke-virtual {v3, v7, v6, v0, v8}, Landroid/app/Notification;->setLatestEventInfo(Landroid/content/Context;Ljava/lang/CharSequence;Ljava/lang/CharSequence;Landroid/app/PendingIntent;)V
  
!     const-string v7, "CaptivePortal.Notification"
  
!     invoke-virtual {v4, v7, v11, v3}, Landroid/app/NotificationManager;->notify(Ljava/lang/String;ILandroid/app/Notification;)V
  
      .end local v0           #details:Ljava/lang/CharSequence;
      .end local v1           #icon:I
--- 886,894 ----
  
      invoke-virtual {v3, v7, v6, v0, v8}, Landroid/app/Notification;->setLatestEventInfo(Landroid/content/Context;Ljava/lang/CharSequence;Ljava/lang/CharSequence;Landroid/app/PendingIntent;)V
  
!     iget-object v7, p0, Landroid/net/CaptivePortalTracker;->mContext:Landroid/content/Context;
  
!     invoke-static {v7, v2}, Landroid/net/CaptivePortalTracker$Injector;->showLogin(Landroid/content/Context;Landroid/content/Intent;)V
  
      .end local v0           #details:Ljava/lang/CharSequence;
      .end local v1           #icon:I
