*** ./smali/com/android/server/VibratorService.smali	2013-10-12 21:21:57.300389851 +0800
--- /home/santajin/MIUI_42/i9100/temp/new_smali/services.jar.out/./smali/com/android/server/VibratorService.smali	2013-10-12 21:25:07.776399069 +0800
***************
*** 1112,1117 ****
--- 1112,1126 ----
      throw v1
  .end method
  
+ .method public vibrateEx([BLandroid/os/IBinder;)V
+     .locals 0
+     .parameter "params"
+     .parameter "token"
+ 
+     .prologue
+     return-void
+ .end method
+ 
  .method public vibratePattern([JILandroid/os/IBinder;)V
      .locals 9
      .parameter "pattern"
