*** ./smali/android/graphics/drawable/DrawableContainer$DrawableContainerState.smali	2013-10-12 21:22:39.464391891 +0800
--- /home/santajin/MIUI_42/i9100/temp/new_smali/framework.jar.out/./smali/android/graphics/drawable/DrawableContainer$DrawableContainerState.smali	2013-10-12 21:26:05.400401858 +0800
***************
*** 782,787 ****
--- 782,796 ----
      return v3
  .end method
  
+ .method public final getVariablePadding()Z
+     .locals 1
+ 
+     .prologue
+     iget-boolean v0, p0, Landroid/graphics/drawable/DrawableContainer$DrawableContainerState;->mVariablePadding:Z
+ 
+     return v0
+ .end method
+ 
  .method public growArray(II)V
      .locals 3
      .parameter "oldSize"
